.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,152 @@
+      - uses: google-github-actions/auth@v1
+        with:
+          credentials_json: '${{ secrets.GOOGLE_CREDENTIALS }}'
+      - uses: google-github-actions/setup-gcloud@v1
+        with:
+          project_id: zeebe-io
+          install_components: gke-gcloud-auth-plugin, kubectl
+      - uses: google-github-actions/get-gke-credentials@v0.8.2
+        with:
+          cluster_name: ${{ inputs.cluster }}
+          location: ${{ inputs.cluster-region }}
+      - uses: azure/setup-helm@v3
+      - name: Add camunda helm repo
+        run: |
+          helm repo add camunda https://helm.camunda.io
+          helm repo update
+      - name: Deploy benchmark installation
+        run: >
+          helm upgrade --install
+          --namespace ${{ inputs.ref }}
+          --create-namespace
+          ${{ inputs.name }} camunda/camunda-platform
+          -f benchmarks/setup/default/zeebe-values.yaml
+          --set zeebe.image.repository=gcr.io/zeebe-io/zeebe
+          --set zeebe.image.tag=${{ inputs.name }}
+          --set zeebe-gateway.image.repository=gcr.io/zeebe-io/zeebe
+          --set zeebe-gateway.image.tag=${{ inputs.name }}
+      - name: Deploy and configure additional benchmark components
+        uses: actions/github-script@v6
+        with:
+          # language=javascript
+          script: |
+            const namespace = context.payload.inputs.name
+            const load = context.payload.inputs.load
+            const components = load.split(", ")
+            for (const component of components) {
+              if (component.includes("=")) {
+                const nameAndRate = component.split("=")
+                const name = nameAndRate.at(0)
+                const rate = nameAndRate.at(1)
+                const manifest = `benchmarks/setup/default/${name}.yaml`
+                await exec.exec('kubectl', ['delete',
+                  `--ignore-not-found=true`,
+                  `--namespace=${namespace}`,
+                  'configmap', `${name}-config`])
+                await exec.exec('kubectl', ['create',
+                  `--namespace=${namespace}`, 'configmap', `${name}-config`,
+                  `--from-literal=CONFIG_FORCE_app_starter_rate=${rate}`,
+                  `--from-literal=CONFIG_FORCE_app_brokerUrl=${namespace}-zeebe-gateway:26500`])
+                await exec.exec(`kubectl apply -n ${namespace} -f ${manifest}`)
+              } else {
+                const manifest = `benchmarks/setup/default/${component}.yaml`
+                await exec.exec('kubectl', ['delete',
+                  `--ignore-not-found=true`,
+                  `--namespace=${namespace}`,
+                  'configmap', `${component}-config`])
+                await exec.exec('kubectl', ['create',
+                  `--namespace=${namespace}`,
+                  'configmap', `${component}-config`,
+                  `--from-literal=CONFIG_FORCE_app_brokerUrl=${namespace}-zeebe-gateway:26500`])
```

Comment:
What about just creating the configmap in the default benchmark folder and replacing there the keys. Then you can also just use `k apply -f` and do not neet do delete and create plus there is only one config map for all.

Reply:
Then we can also skip the config map stuff and do what we did before, just `sed`ing the kubernetes manifests. I wanted to not do any fs manipulation since it's tricky to keep track of the current state and so forth.

> plus there is only one config map for all

but we need different config maps for each component because they run with different rates :thinking:

Reply:
> Then we can also skip the config map stuff and do what we did before, just seding the kubernetes manifests. I wanted to not do any fs manipulation since it's tricky to keep track of the current state and so forth.

Not sure whether I fully understand what you mean. But yes `sed` would my preferred way :sweat_smile: 

>  but we need different config maps for each component because they run with different rates 

Actually you just need different keys in the config map, which you can update/replace.

Reply:
I like envsubst instead of sed, but it's less explicit admittedly.

Reply:
To be honest I don't see the benefit of doing it this way but I also don't want to block if you both prefer it. Since the workflow is only an addition and there are (almost) no changes to the existing scripts and manifests, I'd suggest merging this as is and tackling it in a follow up when we write the entire benchmark deploy stuff in a script that works locally and in GHA.

Reply:
Sorry if I block you here, my point is only that I don't want to have multiple ways of doing things, and of course, it would be great if we have something everyone understands (I feel uncomfortable with javascript) and also can use (run locally). If we have this kind of duplication or different ways of doing things this is often a point where things diverge or fail unexpectedly. Are we sure that we continue to work on this after we merged this PR? Right now it's not clear to me why we can't just reuse the stuff we already have in place. 

I guess if @megglos or @npepinpe has a different opinion than me we can also just go forward.

BTW I still think it is great that you did this, don't get me wrong. :bow:

Reply:
As agreed:
- @oleschoenburg will announce the suggested switch in #ask-zeebe to using the workflows to create benchmarks going forward + summarises what changed towards the "old" scripts (e.g. no commits created etc.)
- if we go for merging it we should remove the old scripts in an immediate follow-up to enforce the usage and not have two ways for doing the same thing